### PR TITLE
Follow merge_from_artifact interface update

### PR DESCRIPTION
**Pull request by potsbo**
### Problem

https://github.com/AltimateAI/vscode-dbt-power-user/issues/1140

### Solution

Describe the implemented solution. Add external references if needed.

### Screenshot/Demo

A picture is worth a thousand words. Please highlight the changes if applicable.

### How to test

- Steps to be followed to verify the solution or code changes
- Mention if there is any settings configuration added/changed/deleted

## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

**Comment by mdesmet** [comment]
Feel free to review the code or test it in your environment and let us know if it works.

File: dbt_core_integration.py [hunk 1]
```diff
@@ -480,15 +487,9 @@ def safe_parse_project(self) -> None:
             self.write_manifest_artifact()
 
             if self.defer_to_prod:
-                with open(self.defer_to_prod_manifest_path) as f:
-                    manifest = WritableManifest.from_dict(json.load(f))
```
Comment (potsbo) [inline dbt_core_integration.py:484]: Without this change, the following error occurs in my environment

> There is a problem in your dbt project. Compilation failed: 'Model' object has no attribute 'is_ephemeral'

**Comment by potsbo** [comment]
@mdesmet wasn't aware of that you were also working on that. Thanks!
Let me take a look.

**Comment by potsbo** [comment]
@mdesmet #1147 perfectly fixes my problem reported at #1140 👍
looking forward to see that merged. thanks!

**Comment by potsbo** [comment]
fixed in #1147

**Pull request closed without merging**